tests/test_auth.py

```diff
@@ -18,3 +18,11 @@ def test_login_updates_global_credentials(self):
         # Verify that the global variable contains the updated credentials
         self.assertEqual("lastplacelarry", get_kaggle_credentials().username)
         self.assertEqual("some-key", get_kaggle_credentials().key)
+
+    def test_set_kaggle_credentials_raises_error_with_empty_username(self):
+        with self.assertRaises(ValueError):
```

Comment:
Can you test e2e from kagglehub.login?

Reply:
Instead of calling `set_kaggle_credentials` here, call:

```
with mock.patch("builtins.input") as mock_input:
            mock_input.side_effect = ["", "some-key"]
            kagglehub.login()
```